### ISSUE TITLE
[BUG] [RHEL | Azure] download-requirements.sh fails when using RHEL image attached to EUS repo

### ISSUE THREAD

**Issue by plirglo** [epiphany-platform/epiphany#1465]
**Describe the bug**
I Wanted to download requirements for offline use on RedHat 7.7 (Azure) but download-requirements.sh script exited with error:
```
ERROR: RHEL yum repository not found, pattern was: rhel-7-server-extras-rpms|rhui-REGION-rhel-server-extras|rhui-rhel-7-server-rhui-extras-rpms
```

**To Reproduce**
In docker container:
1. execute `epicli prepare --os redhat-7`
On RedHat machine:
1. execute: `sudo ./download-requirements.sh /usr/tmp/requirements/`

**Expected behavior**
Script should download requirements to pointed location

**Config files**
```
$ ls -l /etc/yum.repos.d/
total 24
-rw-r--r--. 1 root root  2424 Oct 18  2019 docker-ce.repo
-rw-r--r--. 1 root root   358 May 28 11:24 redhat.repo
-rw-r--r--. 1 root root 12564 May 14  2019 rh-cloud-eus.repo

$ cat /etc/*release
NAME="Red Hat Enterprise Linux Server"
VERSION="7.7 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.7"
PRETTY_NAME="Red Hat Enterprise Linux Server 7.7 (Maipo)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.7:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.7
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.7"
Red Hat Enterprise Linux Server release 7.7 (Maipo)
Red Hat Enterprise Linux Server release 7.7 (Maipo)
```

**OS (please complete the following information):**
 - OS: RHEL7.7

**Cloud Environment (please complete the following information):**
 - Cloud Provider: Azure

**Additional context**
```
$ sudo ./download-requirements.sh /usr/tmp/requirements/
Process with pid: 38779 not found
Removing file: /var/run/download-requirements.pid
PID is: 114653, creating file: /var/run/download-requirements.pid
Backuping /etc/yum.repos.d/ to /home/operations/shared/build/prepare_scripts/download-requirements.sh-yum-repos-backup-tmp-do-not-remove.tar
tar: Removing leading `/' from member names
/etc/yum.repos.d/
/etc/yum.repos.d/redhat.repo
/etc/yum.repos.d/rh-cloud-eus.repo
/etc/yum.repos.d/docker-ce.repo
tar: Archive contains file names with leading prefixes removed.
tar: Verification may fail to locate original files.
Verify etc/yum.repos.d/
Verify etc/yum.repos.d/redhat.repo
Verify etc/yum.repos.d/rh-cloud-eus.repo
Verify etc/yum.repos.d/docker-ce.repo
Backup done
Package yum-utils already installed
Package wget already installed
Package curl already installed
Failed to set locale, defaulting to C
ERROR: RHEL yum repository not found, pattern was: rhel-7-server-extras-rpms|rhui-REGION-rhel-server-extras|rhui-rhel-7-server-rhui-extras-rpms
```

**Comment by plirglo** [comment]
Also having the same error with online installation.

Machine config:
```
specification:
  storage_image_reference:
    publisher: RedHat
    offer: RHEL
    sku: 7.7
    version: "7.7.2020052922"
```

**Comment by plirglo** [comment]
After investigation i found that for Red Hat Azure images we don't support images which are:
- RedHat:RHEL:7-RAW-CI:7.7.2019081601
- have lvm partitioning
- are connected to EUS repositories

see images description:
https://docs.microsoft.com/pl-pl/azure/virtual-machines/workloads/redhat/redhat-images

In this task extend repository parser to handle EUS repositories

**Comment by to-bar** [comment]
Successfully tested `RedHat:RHEL:7-RAW-CI:7.6.2019072418` so it looks like SKU `7-RAW-CI` is also supported.

**Comment by seriva** [comment]
Closed as we are moving to RHEL8 with 2.0. If it comes up again we can create a new issue.
